.eslintrc

```diff
@@ -72,7 +72,9 @@
 
     "react/jsx-closing-tag-location": "warn",
 
-    "import/extensions": "error",
+    "import/extensions": "off",
+
+    "import/no-unresolved": "off",
```

Comment:
These rules should be turned back on.
To prevent issues we should implement eslint overrides config for typescript specific linting rules.
See an example of that here
https://gitlab.uncinc.nl/pub/eslint-config/-/blob/master/index.js#L40